src/js/components/AddShabadButton.tsx

```diff
@@ -48,7 +48,8 @@ export const AddShabadButton: React.FC<FCProps> = ({ shabad: { verseId, shabadId
       data-verse={verse}
       onClick={onClick}>
       <AddIcon />
-      Add
+      <div className="show-on-desktop">
```

Comment:
At homepage search, Autocomplete not works if we add that (it's weird), I will fix it on my end that portion.

Reply:
Good catch! Yes, please fix on your end then.